server/services/communityService.ts

```diff
@@ -1,34 +1,28 @@
-import type HmppsAuthClient from '../data/hmppsAuthClient'
 import CommunityApiClient from '../data/communityApiClient'
 import ProbationSearchApiClient from '../data/probationSearchApiClient'
 import { OffenderDetail, SearchDto } from '../@types/probationSearchApiClientTypes'
 import { CommunityApiManagedOffender, CommunityApiStaffDetails } from '../@types/communityClientTypes'
 
 export default class CommunityService {
-  constructor(private readonly hmppsAuthClient: HmppsAuthClient) {}
+  constructor(
+    private readonly communityApiClient: CommunityApiClient,
+    private readonly probationSearchApiClient: ProbationSearchApiClient
+  ) {}
 
   async getStaffDetail(deliusUsername: string): Promise<CommunityApiStaffDetails> {
-    // Important: No username is passed into the getSystemClientToken for the community API
-    const token = await this.hmppsAuthClient.getSystemClientToken()
-    return new CommunityApiClient(token).getStaffDetailByUsername(deliusUsername)
+    return this.communityApiClient.getStaffDetailByUsername(deliusUsername)
```

Comment:
This is the bit which skipped the username parameter when getting a system token for the community api.